docs/en/siem/detections/detection-engine-intro.asciidoc

```diff
@@ -162,6 +162,9 @@ For *all* deployments (on-premises and hosted):
 ** Write permissions for the `.siem-signals-<space name>` index, such as 
 `create` `create_doc`, `write`, `index`, and `all`
 (see {ref}/security-privileges.html#privileges-list-indices[Indices privileges]).
+** To create {ml} rules, you must have the
```

Comment:
While this is listed under "To create and modify detection rules..." I worry that it might be confusing to call out creation specifically here. I would recommend changing it to either:

```suggestion
** To create or modify {ml} rules, you must have the
```
or
```suggestion
** For {ml} rules, you must have the
```

Reply:
The outcome is...

** To create or modify {ml} rules, you must have the

Thanks @rylnd